__tests__/app.test.js

```diff
@@ -83,6 +83,7 @@ describe("GET: /api/articles/:article_id", () => {
           topic: "mitch",
           created_at: expect.any(String),
           votes: 100,
+          comment_count: expect.any(Number),
```

Comment:
Ideally, you shouldn't be amending past tests. I'd advice using .toMatchObject next time as it won't fail if other properties are added 👍

Reply:
Sure will look into it.Thanks